spec/rspec/core/bisect/runner_spec.rb

```diff
@@ -213,19 +213,75 @@ def repro_command_from(ids)
       open3_method = Open3.respond_to?(:capture2e) ? :capture2e : :popen3
       open3_method = :popen3 if RSpec::Support::Ruby.jruby?
 
+      def called_environment
+        @called_environment
+      end
+
+      if open3_method == :capture2e
+        RSpec::Matchers.define :have_invoked_command_with_env do |command, environment|
+          match do |block|
+            block.call
+
+            expect(Open3).to have_received(open3_method).with(environment, command)
+          end
+
+          supports_block_expectations
+        end
+      elsif open3_method == :popen3
+        RSpec::Matchers.define :have_invoked_command_with_env do |command, environment|
+          match do |block|
+            block.call
+
+            expect(Open3).to have_received(open3_method).with(command)
+            expect(called_environment).to include(environment)
+          end
+
+          supports_block_expectations
+        end
+      end
```

Comment:
Oh - yeah, that's an artifact of switching to the non-block style and back again, will change it now.